test/test_spawn_page.py

```diff
@@ -46,7 +46,7 @@ async def test_spawn_page(app):
     """Test display of spawn page and check embedded SLURM resources info"""
     with mock.patch.dict(app.users.settings, {"spawner_class": MOSSMockSimple}):
         cookies = await app.login_user("jones")
-        r = await get_page("spawn", app, cookies=cookies)
+        r = await request(app, "GET", "spawn", cookies=cookies)
```

Comment:
Using a unique function to make POST and GET requests ease parameterized tests (see below).